ios/Exponent/Kernel/Services/EXUpdatesManager.m

```diff
@@ -167,7 +167,7 @@ - (void)updatesModule:(id)scopedModule
   EXUpdatesDatabaseManager *databaseKernelService = [EXKernel sharedInstance].serviceRegistry.updatesDatabaseManager;
   EXAppLoader *appLoader = [self _appLoaderWithScopedModule:scopedModule];
 
-  EXUpdatesRemoteAppLoader *remoteAppLoader = [[EXUpdatesRemoteAppLoader alloc] initWithConfig:appLoader.config database:databaseKernelService.database directory:databaseKernelService.updatesDirectory completionQueue:completionQueue];
+  EXUpdatesRemoteAppLoader *remoteAppLoader = [[EXUpdatesRemoteAppLoader alloc] initWithConfig:appLoader.config database:databaseKernelService.database directory:databaseKernelService.updatesDirectory launchedUpdate:nil completionQueue:completionQueue];
```

Comment:
Tangentially I found no references to `didRequestBundleWithCompletionQueue:` in our codebase. Looks like it was when an older version of EXUpdates (in ExpoKit) called into it. Is this whole method still needed?

Reply:
That's a good point, I think that is the case. I'd still rather keep this change as is (so the project compiles) and remove it in a separate PR to reduce risk, especially since we're so near to an SDK release.